cbor_event/README.md

```diff
@@ -9,14 +12,56 @@ This crate has 0 dependency (and should not need any in the future). This
 is a design choice in order to guarantee as much as possible of the
 compatibility across multiple platform.
```

Comment:
While some libraries provide an intermediate type representation, @@ -9,14 +12,56 @@ This crate has zero dependencies (and should not need any in the future). This is a design choice in order to guarantee as much compatibility as possible across multiple platforms.